content/pages/nuttx/getting-started/index.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Getting Started with NuttX and ESP32 Family"
+date: 2024-06-20T16:54:43-03:00
+tags: ["NuttX"]
+authors:
+    - "tiago-medicci"
+    - "eren-terzioglu"
+    - "filipe-cavalcanti"
+---
+
+## Getting Started with NuttX and ESP32 Family
+
+### Apache NuttX RTOS
```

Comment:
This article is structured as follows

- Intro to NuttX
- Supported Espressif SoCs
- Gradual dive into setting up the environment and building the application without explicitly mentioning it in any higher level heading

Suggest making the structure more explicit.'

Another important thing is to provide an introduction to the article announcing what the user will do or learn throughout this article. Currently, this part is missing.

Reply:
I completely changed the sections considering this comment. Please double-check the new structure.

> Another important thing is to provide an introduction to the article announcing what the user will do or learn throughout this article. Currently, this part is missing.

Added in https://github.com/espressif/developer-portal/pull/45/files#diff-024a8213ef540f9f04acbb2d49638d2ba31f7dce21b9c0e8e0f613238bbac90fR39